dev/ToastNotifications/ToastActivationInfo.cpp

```diff
@@ -6,18 +6,25 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
 {
     winrt::Microsoft::Windows::ToastNotifications::ToastActivationInfo ToastActivationInfo::CreateFromActivationGuid(winrt::guid const& taskClsid)
     {
-        throw hresult_not_implemented();
+        THROW_HR_IF_MSG(E_NOTIMPL, !AppModel::Identity::IsPackagedProcess(), "Not applicable for unpackaged applications");
+
+        return winrt::make<ToastActivationInfo>(taskClsid, nullptr);
```

Comment:
REcommend having 2 specialized constructors: One that takes in a GUID and another that takes in ToastAssets